examples/knative/go/kn-go-nsx-tag-sync/function.yaml

```diff
@@ -60,7 +60,7 @@ spec:
   broker: default
   filter:
     attributes:
-      subject: com.vmware.cis.tagging.attach
+      subject: com.vmware.vsphere.com.vmware.cis.tagging.attach.v0
```

Comment:
```suggestion
      type: com.vmware.vsphere.com.vmware.cis.tagging.attach.v0
```

same below

Reply:
Updated

Reply:
You need to change `subject` to `type` ;)